### PR TITLE
adding json generator reference

### PR DESCRIPTION
**Pull request by memsharded**
Adding reference section

### CODE REVIEW

**Comment by dmerejkowsky** [comment]
@memsharded I think you meant to create a pull request on your repo (conan-io/docs) :P

**Comment by memsharded** [comment]
No, this is totally intended. I am trying to complete your open PR: https://github.com/conan-io/docs/pull/552. You can see I branched from your branch.

This way is nicer, all the docs from this feature come from a single source (your repo), and you are most likely getting attribution for your contribution in case that the commits are squashed.

**Comment by dmerejkowsky** [comment]
Oh, OK. That's a bit unusual for me but it makes sense.

File: reference/generators/json.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          "name": "Poco",
+          "version": "1.7.8p3",
+          "...": "..."
+        }
+      ],
+    }
+
+
+
+The generated ``conanbuildinfo.json`` file is a json file with the following fields:
```
Comment (dmerejkowsky) [inline reference/generators/json.rst]: maybe 'keys' instead of 'fields' ? Seems more common.

File: reference/generators/json.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+dependencies
+-------------
+
+The dependencies is a list, with each item belonging to one dependency, and each one with the following fields:
```
Comment (dmerejkowsky) [inline reference/generators/json.rst]: Maybe explain that the order matters ? People may be used to the dependencies being a map and not an array.

**Pull request merged**